spec/features/pilot/step_1_understand_service_spec.rb

```diff
@@ -16,6 +16,8 @@
     visit "/"
     should_see_what_the_service_can_provide
     should_see_what_information_i_will_need_to_provide
+    when_i_choose_to_use_the_service
+    then_i_should_find_myself_on_the_first_step_of_the_service
```

Comment:
Following on from earlier comment on step numbering, this confounds things for me. This entire spec is about "Step 1", but part way through the expectations of step 1, we expect to arrive at... the first step (of the service)

Reply:
Yes fair point! This was confusing!

I have reworked the commits so that this is now referred to as "Step 0: start page" throughout.

The source of the confusion is that we're currently using the term "step" for 2 different concepts:

1. the discrete stages in the multi-form questionnaire or wizard (e.g. "step 1: destination" and "step 2: dates")
2. the individual lines in the acceptance tests (e.g `when_i_choose_to_use_the_service`, and `then_i_should_find_myself_on_the_first_step_of_the_service `)

Perhaps 1) should be renamed "stage" and the 2) can continue to be "step" -- which is an established term in various BDD systems?

Reply:
Makes sense to me, even though I'm used to forms having steps rather than stages in RODA!